docs/docs/core/08-events.md

```diff
@@ -0,0 +1,22 @@
+---
+sidebar_position: 1
+---
+# Events
+
+:::note Synopsis
+`Event`s are objects that contain information about the execution of the application. They are mainly used by service providers like block explorers and wallet to track the execution of various messages and index transactions.
+:::
+
+## Default Events
+
+There are a few events that are automatically emitted for all messages, directly from `baseapp`.
+
+* `message.action`: The name of the message type.
+* `message.sender`: The address of the message signer.
+* `message.module`: The name of the module that emitted the message.
```

Comment:
As far as I know, the `message.module` is not emitted every time. If then, how about adding an explanation that it may not be added?

Reply:
You may refer to:
https://github.com/Finschia/finschia-sdk/blob/44c0a1efa1353cd2a848577c43adceb7a41b049b/docs/docs/core/08-events.md?plain=1#L20-L21